### PR TITLE
Support PROPERTIES=live as live ebuild indicator

### PR DESCRIPTION
**Pull request by mgorny**

### CODE REVIEW

**Comment by mgorny** [comment]
CC @ulm

File: src/pkgcore/ebuild/ebuild_src.py [hunk 1]
```diff
@@ -314,7 +314,8 @@ def mandatory_phases(self):
 
     @property
     def live(self):
-        return bool(const.VCS_ECLASSES.intersection(self.inherited))
+        return ('live' in self.properties or
+                bool(const.VCS_ECLASSES.intersection(self.inherited)))
```
Comment (ulm) [inline src/pkgcore/ebuild/ebuild_src.py]: Why keep the eclasses here?

Comment (radhermit) [reply src/pkgcore/ebuild/ebuild_src.py]: In part because pkgcheck was using it as well for tests.

**Comment by radhermit** [comment]
Thanks, merged with some changes.

**Pull request closed without merging**